src/main/scala/tech/cryptonomic/conseil/michelson/JsonToMichelson.scala

```diff
@@ -0,0 +1,15 @@
+package tech.cryptonomic.conseil.michelson
```

Comment:
As Michelson is a Tezos-specific language could we move this package into the `tezos` package?